app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,55 @@
+class InventoriesController < ApplicationController
+  before_action :set_inventory, only: %i[show destroy]
+
+  # GET /inventories or /inventories.json
+  def index
+    @inventories = Inventory.where(user_id: current_user.id)
+  end
+
+  # GET /inventories/1 or /inventories/1.json
+  def show
+    @food_inventories = FoodInventory.includes(:food).where(inventory_id: params[:id])
+  end
+
+  # GET /inventories/new
+  def new
+    @inventory = Inventory.new
+  end
+
+  # POST /inventories or /inventories.json
+  def create
+    @inventory = Inventory.new(inventory_params.merge(user: current_user))
+
+    respond_to do |format|
+      if @inventory.save
+        format.html { redirect_to inventories_url, notice: 'Inventory was successfully created.' }
+        format.json { render :show, status: :created, location: @inventory }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @inventory.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /inventories/1 or /inventories/1.json
+  def destroy
+    @inventory.destroy!
+
+    respond_to do |format|
+      format.html { redirect_to inventories_url, notice: 'Inventory was successfully destroyed.' }
+      format.json { head :no_content }
+    end
+  end
```

Comment:
- [ ] @Team it's always a good practice, to have success and failure paths for creating, updating, and destroying actions, whenever an action fails and our app doesn't have the option to refer the user it might seem like the app is broken.